data/swtpm/ssh_port_chk_script

```diff
@@ -3,10 +3,11 @@ set -euo pipefail
 
 ip_addr=""
 for (( i = 0; i < 200; i++ )); do
```

Comment:
If still not connected ssh after 200 seconds, what program should do?

Reply:
timeout=200 should be enough :)

Reply:
I suggest you add RETRY=1 in job setting, I can always passed the test after re-run:)

Reply:
> timeout=200 should be enough :)

Sure, I'll change it back.